components/project-list/project-list.jsx

```diff
@@ -66,15 +88,6 @@ export default React.createClass({
       return entries.sort((a,b) => {
         return bookmarkedIdArray.indexOf(a.id.toString()) > bookmarkedIdArray.indexOf(b.id.toString());
```

Comment:
Is it worth doing this sorting on Pulse API side instead?

e.g, `/entries/?ids=3,1,5` will return entry no.3 first, followed by no.1, and lastly no.5.